tariff/tariffs.go

```diff
@@ -0,0 +1,19 @@
+package tariff
+
+import (
+	"github.com/evcc-io/evcc/api"
+)
+
+type Tariffs struct {
```

Comment:
Was tut das?

Reply:
Das Struct? You tell me :D Ich adaptiere hier die Struktur der anderen Module. Tariff ist die Repräsentation des Konfigurationsobjekts.
Was würdest du anders machen?

Reply:
Ich hab jetzt verstanden, dass es 2 Tarife sammelt ;)